src/map/skill.cpp

```diff
@@ -18244,7 +18244,7 @@ int32 skill_check_pc_partner(map_session_data *sd, uint16 skill_id, uint16 *skil
 		return is_chorus ? MAX_PARTY : 99; //As if there were infinite partners.
 
 	if (cast_flag) {	//Execute the skill on the partners.
-		map_session_data* tsd;
+		map_session_data* tsd = nullptr;
```

Comment:
```suggestion
		map_session_data* tsd;
```
Can be reverted now again, since it is now always initialized.